docs/source/create_an_experiment_(development).rst

```diff
@@ -0,0 +1,723 @@
+the experiment and send the user to the Dallinger questionnaire page. If
+the rejection status is not "403", that means something unexpected
+happened, and we need to raise a Dallinger error, effectively ending the
+experiment.
+
+Now let's discuss the `get_info` function mentioned above, which is
+called when the experiment first calls the `create_agent` function:
+
+.. code-block:: javascript
+
+  var get_info = function() {
+    dallinger.getReceivedInfos(my_node_id)
+    .done(function (resp) {
+      var story = resp.infos[0].contents;
+      $("#story").html(story);
+      $("#stimulus").show();
+      $("#response-form").hide();
+      $("#finish-reading").show();
+    })
+    .fail(function (rejection) {
+      console.log(rejection);
+      $('body').html(rejection.html);
+    });
+  };
+
+Remember that in the Python code above, in the `add_node_to_network`
+method, we looked for the participant's parent, and then called its
+`transmit` method, followed by the node's own `receive` method. This
+transmits the parent node's info to the new node. The Javascript `get_info`
+function tries to get that info by calling `dallinger.getReceivedInfos` with
+the node id that we saved after successfully calling `dallinger.createAgent`.
+
+For the first participant, this info will contain the text generated by the
+source we defined above. That is, the full text of one of the stimulus
+stories, chosen at random. The second participant will get the text as
+recalled by the first participant, and so on. The last participant will
+likely have a much different text to work with than the first.
+
+Once `get_info` gets the text, it puts it in the `story` textarea, and
+shows it to the user, by displaying the `stimulus` div. Then it makes sure
+the `response-form` is not visible, and shows the `finish-reading` button.
+
+If anything fails, we log the rejection message to the console, and show
+the error to the user.
+
+The experiment templates
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The experiment uses regular dallinger templates for the ad page and
+consent form. It does define its own layout, as an example of how to
+include dependencies. Here's the full ``layout.html`` template:
+
+::
+
+	{% extends "base/layout.html" %}
+
+	{% block title -%}
+		Bartlett 1932 Experiment
+	{%- endblock %}
+
+	{% block libs %}
+		<script src="/static/scripts/store+json2.min.js" type="text/javascript"> </script>
+		{{ super() }}
+		<script src="/static/scripts/experiment.js" type="text/javascript"> </script>
+	{% endblock %}
+
+The only important part if the layout template is the `libs` block. Here you
+can add any Javascript dependencies that your experiment needs. Just place
+them in the experiment's ``static`` directory, and they will be available for
+linking from this page.
+
+Note how we load everything else before the ``experiment.js`` file that
+contains our experiment code (The `super` call brings up any dependencies
+defined in the base layout).
```

Comment:
Why are we putting `store+json2.min.js` before the base dependencies?

Reply:
store+json2.min.js shouldn't be included here at all. It will be pulled in with the `super()` call, because it's part of Dallinger.